.github/workflows/golangci-lint.yml

```diff
@@ -21,9 +21,9 @@ jobs:
       - uses: actions/checkout@v3
```

Comment:
(just had to put the comment somewhere)

Can we please also revert the changes from this [PR](https://github.com/kubernetes-sigs/cluster-api/pull/6351/files) to `internal/webhooks/cluster_test.go` and `internal/webhooks/clusterclass_test.go`

Reply:
@sbueringer I reverted and changed the code since the function was moved in this commit: https://github.com/kubernetes-sigs/cluster-api/commit/ce89e60370ba49bd3599f72d80920ac95bc0f994 I hope this is what you wanted.

Reply:
Looks perfect. Thx!